shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -983,6 +994,19 @@ && shouldSetCollectionInfo(semanticsNode)) {
     return result;
   }
 
+  private boolean isImportant(SemanticsNode node) {
+    if (node.hasFlag(Flag.SCOPES_ROUTE)) {
+      return false;
+    }
+
+    if (node.getValueLabelHint() != null) {
+      return true;
+    }
+
+    // Return true if it has any user action.
```

Comment:
I think "it" is the node if so can you use the node instead of it?
```suggestion
    // Return true if the node has had any user action (not including system actions) 
```